chaos_kitten/validators/profile_validator.py

```
@@ -0,0 +1,143 @@
+        except Exception as e:
+            report.is_valid = False
+            report.errors.append(f"Error reading file: {e}")
+            return report
+            
+        if not isinstance(content, dict):
+            report.is_valid = False
+            report.errors.append("Profile root must be a dictionary/map")
+            return report
+
+        # Check required fields
+        for field in self.REQUIRED_FIELDS:
```

Ruff (0.15.0)

[error] Import `field` from line 1 shadowed by loop variable

(F402)